.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
> If you only ship the development build 

If you do so, you are in a lot more trouble because of React. We can safely ignore it.